src/common/analytics/Analytics.ts

```diff
@@ -51,57 +58,74 @@ export class ProductAnalytics {
     selectedNetwork$.pipe(first()).subscribe(cb);
   }
 
-  public firstLaunch({
-    active_network,
-    active_locale,
-    active_theme,
-    cohort_date,
-    cohort_month,
-    cohort_day,
-    cohort_version,
-    cohort_year,
-  }: {
-    active_network: SupportedNetworks;
-    active_locale: string;
-    active_theme: 'light' | 'dark' | 'system';
-    cohort_date: string;
-    cohort_day: number;
-    cohort_month: number;
-    cohort_year: number;
-    cohort_version: string;
-  }): void {
-    this.event(
-      ANALYTICS_EVENTS.FIRST_LAUNCH,
-      {},
-      {
-        set: {
+  public init: Initializer = () => {
```

Comment:
`this.analyticsSystem` after `init()` should contain only initialized systems because any analytics system  can give an error after you will try to use sendEvent before initialization.